templates/form/input--submit-button.html.twig

```diff
@@ -12,4 +12,4 @@
  * @see template_preprocess_input()
  */
 #}
-<button{{ attributes }}>{{ attributes.value|raw }}</button>{{ children }}
\ No newline at end of file
+<button{{ attributes.removeClass('btn-sm') }}>{{ attributes.value|raw }}</button>{{ children }}
```

Comment:
Can the btn-sm /lg be set on the Barrio UI too? Or is that fixed?

Reply:
@DiegoPino not sure your question. I am not touching the barrio theme code at all.